gradle-baseline-java/src/main/groovy/com/palantir/baseline/extensions/BaselineErrorProneExtension.java

```diff
@@ -38,6 +38,7 @@
             "PreferListsPartition",
             "PreferSafeLoggableExceptions",
             "PreferSafeLoggingPreconditions",
+            "RawTypes",
```

Comment:
This isn't necessary because the RawTypes check does not provide a suggested fix, this list enumerates the checks that we automatically fix by default.

I wonder how many projects this will impact, we could implement a automated fix to apply `?` as a replacement. I'm sure there are edge cases that it won't fix, but it should make most cases slightly better. The counterargument is that most of these should be replaced with more precise type parameters that humans are better equipped to write.

Reply:
Removed from this list.

I'd rather leave existing cases marked as warnings for human review than have them replaced by `?` or `Object`.

Reply:
Sounds good